core/lib/zksync_core/src/block_reverter/mod.rs

```diff
@@ -351,9 +351,9 @@ impl BlockReverter {
 
     async fn get_l1_batch_number_from_contract(&self, op: AggregatedActionType) -> L1BatchNumber {
         let function_name = match op {
-            AggregatedActionType::Commit => "getTotalBlocksCommitted",
-            AggregatedActionType::PublishProofOnchain => "getTotalBlocksVerified",
-            AggregatedActionType::Execute => "getTotalBlocksExecuted",
+            AggregatedActionType::Commit => "getTotalBatchesCommitted",
+            AggregatedActionType::PublishProofOnchain => "getTotalBatchesVerified",
+            AggregatedActionType::Execute => "getTotalBatchesExecuted",
```

Comment:
Let's change it later so that if we need rollback batches before the upgrade we can do that. Old getters are not removed, so it will work for all batches with them